source/AST/DecodeRecord.hpp

```diff
@@ -99,20 +99,20 @@ decodeRecord(
     return llvm::Error::success();
 }
 
-// vector<RefWithAccess>
+// vector<MemberRef>
 inline
 llvm::Error
 decodeRecord(
     Record const& R,
-    std::vector<RefWithAccess>& f,
+    std::vector<MemberRef>& f,
     llvm::StringRef blob)
 {
-    constexpr auto RefWithAccessSize =
+    constexpr auto MemberRefSize =
```

Comment:
I would have just called it `Size` but w/e ...

Reply:
I just used replace all to change the names :) I don't mind what the name is either way